mysql/datadog_checks/mysql/statement_samples.py

```diff
@@ -479,16 +495,22 @@ def _collect_plan_for_statement(self, row):
             return None
         self._explained_statements_cache[query_cache_key] = True
 
-        plan = None
+        plan, explain_err_code, err = None, None, None
```

Comment:
For clarity, since this function is quite long, maybe instead of `err` call the variable `explain_err`? 

Also for clarity we should maintain the same variable names for the same things in other functions. So in `_explain_statement` instead of `db_explain_error, err` use `explain_err_code, explain_err`